tests/suites/test_suite_psa_crypto_se_driver_hal.data

```diff
@@ -0,0 +1,28 @@
+Register SE driver: good
```

Comment:
`depends_on:MBEDTLS_PSA_CRYPTO_SE_C` for this and all other test cases?

Reply:
It's a dependency of the whole `.function` file.